server/parsec/_parsec_pyi/ids.pyi

```diff
@@ -214,3 +214,20 @@ class BootstrapToken:
     def int(self) -> int: ...
     @property
     def hyphenated(self) -> str: ...
+
+class ShamirRevealToken:
```

Comment:
Is there a reason for tokens to be defined here with ids ?

Reply:
`InvitationToken` is also defined in this file, we curently don't have a specific file for token types

Reply:
cf: https://github.com/Scille/parsec-cloud/issues/7388